src/content/docs/accounts/accounts/automated-user-management/okta-scimsso-application-configuration.mdx

```diff
@@ -80,13 +80,36 @@ Assigning users is done using two different tabs in the app. We recommend having
 
 When you've added one or more groups, you should be able to see the users you've added by going to the [**User management** UI page](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where). 
 
-## Step 5. Optional: Downgrade some users to basic [#downgrade]
+## Step 5. Manage user type [#downgrade]
 
-New Relic users provisioned via an identity provider start out as [full users](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#user-type). If your organization is on [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), these users are billable. To convert users to free [basic users](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#user-type), use the [User management UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles#add-users).
+Once provisioned, your New Relic users start out as [full users](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#user-type). If your organization is on [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), these users are billable. 
+
+To convert users to free [basic users](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#user-type), you have two choices: 
+
+* Manage user type from New Relic using the [User management UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles#add-users), OR
+* Manage user type from Okta (described below). 
+
+To manage your users' user type from Okta: 
+
+1. Go to the [authentication domain UI](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#manage-user-type-scim) and click **Enable Manage user type with SCIM**. Note that when this is enabled, you can’t manage user type from the New Relic UI and can only manage it from Okta.
+2. Next, you'll configure Okta to be able to send a new attribute `nrUserType`. Steps: 
```

Comment:
Step 2 onward are all in Okta right? I think it would be helpful to make that more clear.

Reply:
Also, I don't think you need the "Steps:" bit